src/WiFiMgr.cpp

```diff
@@ -31,7 +31,7 @@
 //-----------------------------------------------------------------------------
 // Create secrets.h with a #define for SECRETS_SSID and SECRETS_PASS
 // or delete the #include and enter the strings directly below.
-#include "secrets.h"
```

Comment:
This will probably get uncommented since I use it for development.  The Travis build script generates sescrets.h as well so it compiles fine there.